### DISCUSSION TITLE
What is Brunch driver support for Broadcom BCM94350/BCM94352/BCM94360 ?

### DISCUSSION THREAD

**Discussion by acmehax** [sebanc/brunch#1781] [Q&A]
I'm running Intel Haswell tiny form factor desktops with these Broadcom WiFi chips.

Toiled for a few days to finally discover that the brunch 107 stable release of 20221029 either ignores the broadcom_wl kernel config option or does not have the Broadcom drivers within. Thanks to the forum, I discovered that the brunch 107 unstable release of 20221115 does actually pay attention to the broadcom_wl kernel config option and so then Broadcom WiFi works in this release.

So now it looks like it's the same situation with the latest brunch 109 stable release of 20230201. Again, the broadcom_wl kernel config option seems to have no effect and Broadcom WiFi is not functioning with this version.

Is there a place where device driver support for these brunch releases is documented or any other prescribed method to figure it out ?

All three of these Broadcom variations are fully functional with default Linux drivers on the live boot media.

Does brunch use something other than linux drivers when patching in the broadcom_wl package ?

**Comment by tejasraman** [comment]
No... It was broken for some time.
See #1679
